### PR TITLE
Bump react, babel, webpack to latest published versions

### PR DESCRIPTION
**Pull request by adamburmister**
To: @zackargyle 

Updated:
- React to 0.14.3 -- and while I'm at it: Babel to 6.3.13, Webpack to 1.12.9
- Fix up deprecation warnings
- Bump version to v1.1.0 pending release

### CODE REVIEW

File: src/components/PinterestGridWidgetBase.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (zackargyle) [inline src/components/PinterestGridWidgetBase.js]: Is this import needed?

**Comment by zackargyle** [comment]
Looks awesome! Only the one question. Thanks for doing this. :)

File: src/components/PinterestGridWidgetBase.js (hunk 1 above)
Comment (adamburmister) [inline src/components/PinterestGridWidgetBase.js]: Nope! No longer needed. Removing...

**Comment by adamburmister** [comment]
@zackargyle resolved redundant import

**Pull request merged**